plugins/modules/rds_instance.py

```diff
@@ -1631,6 +1646,7 @@ def main():
         monitoring_interval=dict(type="int"),
         monitoring_role_arn=dict(),
         multi_az=dict(type="bool"),
+        multi_tenant=dict(type="bool", default=False),
```

Comment:
```suggestion
        multi_tenant=dict(type="bool", default=False),
```

As a new feature I think this is better defaulting to None rather than False.  Otherwise (I think) it'll try to explicitly set it to False on older versions of botocore...